cluster.go

```diff
@@ -259,6 +261,7 @@ func (c *Cluster) broadcastMetric(m api.Metric) error {
 // push metrics loops and pushes metrics to the leader's monitor
 func (c *Cluster) pushInformerMetrics() {
 	timer := time.NewTimer(0) // fire immediately first
+	retrying := false
```

Comment:
the idea here is that when something is down, this will retry to send metrics every 0.5 secs. This pollutes the logs constantly (which are already full with other messages). Perhaps a compromise is to re-print every 60 retries (30 seconds)?

Reply:
I'll submit a change in that direction.